arcgis-ios-sdk-samples/Layers/Map image layer tables/MapImageLayerTables.storyboard

```diff
@@ -0,0 +1,64 @@
+                                <rect key="frame" x="0.0" y="813" width="414" height="49"/>
+                                <items>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="tPQ-Po-91y"/>
+                                    <barButtonItem enabled="NO" title="Query related spatial features" id="NZ4-Ky-Fcl">
+                                        <connections>
+                                            <action selector="queryFeaturesActions" destination="Y6W-OH-hqX" id="c1X-dV-P2E"/>
+                                        </connections>
+                                    </barButtonItem>
+                                    <barButtonItem style="plain" systemItem="flexibleSpace" id="2YM-Up-BGL"/>
+                                </items>
+                            </toolbar>
+                        </subviews>
+                        <viewLayoutGuide key="safeArea" id="vDu-zF-Fre"/>
+                        <color key="backgroundColor" systemColor="systemBackgroundColor"/>
+                        <constraints>
+                            <constraint firstItem="f1B-6G-Emv" firstAttribute="bottom" secondItem="vDu-zF-Fre" secondAttribute="bottom" id="50V-3Q-7D4"/>
+                            <constraint firstItem="wGx-yp-QOz" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="59w-oX-lfX"/>
+                            <constraint firstItem="f1B-6G-Emv" firstAttribute="leading" secondItem="5EZ-qb-Rvc" secondAttribute="leading" id="A4q-qz-vJs"/>
+                            <constraint firstAttribute="trailing" secondItem="f1B-6G-Emv" secondAttribute="trailing" id="Fbs-Am-vKa"/>
+                            <constraint firstItem="wGx-yp-QOz" firstAttribute="top" secondItem="5EZ-qb-Rvc" secondAttribute="top" id="S3V-Rz-Zuh"/>
+                            <constraint firstAttribute="trailing" secondItem="wGx-yp-QOz" secondAttribute="trailing" id="acU-FP-bEt"/>
+                            <constraint firstItem="f1B-6G-Emv" firstAttribute="top" secondItem="wGx-yp-QOz" secondAttribute="bottom" symbolic="YES" id="fWW-v0-tRT"/>
+                        </constraints>
+                    </view>
+                    <connections>
+                        <outlet property="mapView" destination="wGx-yp-QOz" id="VDV-wE-ONk"/>
+                        <outlet property="queryBarButtonItem" destination="NZ4-Ky-Fcl" id="jh4-Ma-ixn"/>
+                    </connections>
+                </viewController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="Ief-a0-LHa" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="106" y="42"/>
+        </scene>
+    </scenes>
+    <resources>
+        <systemColor name="systemBackgroundColor">
```

Comment:
Should be tiertiarySystemBackgroundColor for root view, default (no color) for map view.